src/ua/javarush/service/EncryptionService.java

```diff
@@ -0,0 +1,104 @@
+package ua.javarush.service;
+
+import ua.javarush.constans.Constans;
+import ua.javarush.constans.EncryptionCommandTypes;
+import java.util.List;
+
+public class EncryptionService {
+    private static final int minTextLengthForCheckAlphabet = 50;
```

Comment:
Constant names use UPPER_SNAKE_CASE: all uppercase letters, with each word separated from the next by a single underscore.